src/Transport/Addressing/Partitioning/Strategies/FailOverNamespacePartitioning.cs

```diff
@@ -11,9 +11,16 @@ public class FailOverNamespacePartitioning : INamespacePartitioningStrategy
 
         public FailOverNamespacePartitioning(ReadOnlySettings settings)
         {
-            if (!settings.TryGet(WellKnownConfigurationKeys.Topology.Addressing.Partitioning.Namespaces, out namespaces) || namespaces.Count != 2)
+            if (!settings.TryGet(WellKnownConfigurationKeys.Topology.Addressing.Partitioning.Namespaces, out namespaces))
             {
-                throw new ConfigurationErrorsException("The 'FailOver' namespace partitioning strategy requires exactly two namespaces to be configured");
+                throw new ConfigurationErrorsException($"The '{nameof(FailOverNamespacePartitioning)}' strategy requires exactly two namespaces to be configured");
```

Comment:
For others exception message transport provides to users the way to fix the issue, reporting snippets or something similar to be more helpful. Should we improve the ex. message here?